components/builder-worker/src/runner/mod.rs

```diff
@@ -315,6 +315,11 @@ impl Runner {
         self.check_cancel(tx).await?;
         let mut section = streamer.start_section(Section::PublishPackage)?;
 
+        if env::var_os("HAB_FUNC_TEST").is_some() {
+            section.end()?;
+            return Ok(());
+        }
+
```

Comment:
rather than repeating these 2 lines, you can wrap the `match` block below in a `if env::var_os("HAB_FUNC_TEST").is_none()`.

Reply:
updated